spec/config/initializers/active_job_logger_patch_spec.rb

```diff
@@ -1,38 +0,0 @@
-require 'rails_helper'
-
-# Covers config/initializers/active_job_logger_patch.rb, which overrides
-# ActiveJob::Logging::LogSubscriber to standardize output and prevent sensitive
-# user data from being logged.
-describe ActiveJob::Logging::LogSubscriber do
```

Comment:
logging has changed some since this was implemented and will need to be redone as part of LG-4369